cluster/ci/config/prow/openshift/deck.yaml

```diff
@@ -82,7 +82,7 @@ objects:
           - --config-path=/etc/config/config.yaml
           - --cookie-secret=/etc/cookie/cookie
           - --github-oauth-config-file=/etc/github/config.json
-          - --github-token-path=/etc/oauth/oauth
+          - --github-token-path=/etc/github/oauth
```

Comment:
So we can keep this one, right? We only want to un-do the change to `oauth-proxy` container

Reply:
Yes. That is what I thought too.

https://github.com/openshift/release/pull/4783#issuecomment-522685976